Sources/FoundationEssentials/Date.swift

```diff
@@ -120,6 +123,10 @@ public struct Date : Comparable, Hashable, Equatable, Sendable {
         return self.timeIntervalSinceReferenceDate + Date.timeIntervalBetween1970AndReferenceDate
     }
 
+    public var timeIntervalSince1601: TimeInterval {
```

Comment:
Can this be internal or package?

Reply:
That is meant to be `internal`, copy-pasta :(.